zerver/lib/bugdown/codehilite.py

```diff
@@ -220,6 +220,6 @@ def extendMarkdown(self, md, md_globals):
         md.registerExtension(self)
 
 
-def makeExtension(configs={}):
-  return CodeHiliteExtension(configs=configs)
+def makeExtension(configs=None):
+  return CodeHiliteExtension(configs=configs or [])
```

Comment:
I think the signature would technically be `Iterable[Tuple[Any, Any]]`.  So any empty iterable works.  But the single call site passes in a list matching that type, so I thought an empty list was more appropriate.  And while an empty dict would work, a dict with anything in it would fail with `ValueError: need more than 1 value to unpack` unless the line `for key, value in configs:` used `configs.items()`.  I just made that mistake for the millionth time a week ago.
I do think the code is a little confusing (e.g. usually `key, value` would imply unpacking a dict).  Maybe we should talk about it a bit more.  You mentioned something about the correct approach being to fix the upstream code, but I wasn't clear who was upstream.